test_runner/Pipfile

```diff
@@ -15,10 +15,13 @@ cached-property = "*"
 
 [dev-packages]
 flake8 = "*"
-mypy = "*"
 # Behavior may change slightly between versions. These are run continuously,
 # so we pin exact versions to avoid suprising breaks. Update if comfortable.
 yapf = "==0.31.0"
+mypy = "==0.910"
+# Non-pinned packages follow.
```

Comment:
flake8 is not pinned to a version either, could you make that more consistent?

Reply:
However, if someone is to run `flake8` and check their code for warnings, I think running the most recent version is better than an arbitrary pinned one.